content/pages/healthcare/health-conditions/mental-health.md

```diff
@@ -29,13 +29,24 @@ If you qualify for VA health care, you can get high-quality mental health servic
 
 </div>
 
-**Are you in crisis? You have options.** [UX/DESIGN NOTE: WE'D LIKE THIS CRISIS CALLOUT TO BE AN ACCORDION PLEASE.]
+<div class="usa-alert usa-alert-warning va-alert">
+  <div class="usa-alert-body">
+	<h5>Are you in crisis? <a id="crisis-expander-link">You have options.</a></h5>
+	<div id="apply-expander-content" class="expander-content expander-content-closed">
```

Comment:
Just to be idiomatic, can we change this `id` to something like `crisis-expander-content`? I originally called it `apply` because it was for expanding the form selection on the education benefits apply page.

We'll also need to change it in the javascript below. 
Change
```js
toggleClass('apply-expander-content', 'expander-content-closed');
```
to
```js
toggleClass('crisis-expander-content', 'expander-content-closed');
```